src/View/SSTemplateParser.peg

```diff
@@ -286,15 +277,17 @@ class SSTemplateParser extends Parser implements TemplateParser
 
         if (isset($sub['Call']['CallArguments']) && isset($sub['Call']['CallArguments']['php'])) {
             $arguments = $sub['Call']['CallArguments']['php'];
-            $res['php'] .= "->$method('$property', [$arguments], true)";
+            $type = ViewLayerData::TYPE_METHOD;
+            $res['php'] .= "->$method('$property', [$arguments], '$type')";
```

Comment:
I would like to keep it for the reasons I've mentioned in a couple of places in this PR:
- https://github.com/silverstripe/silverstripe-framework/pull/11405#discussion_r1811790076
- https://github.com/silverstripe/silverstripe-framework/pull/11405/files#r1811806754

Reply:
Rationale from the two links copied here:

_- I got that idea from twig which does this same thing. It allows to explicitly call methods, or explicitly fetch properties
- Keep a door open for us to use them in future enhancements
- Allow projects and modules to choose to be more strict about how values are accessed in their view layer
- The only reason I didn't use them already is because of pre-existing assumptions about how values will be accessed, e.g. the Field() method in FormField is called when using $Field in a template. This takes precedence over the getField() method it inherits from ModelData even though we're accessing it as a property and not as a method._

I'm not really sure how including this param/arg in this PR really helps with points 2 & 3 here?

Right now, ViewLayerData::getRawDataValue() will just ignore $type, so ... ` projects and modules to choose to be more strict` doesn't work right now?

`Keep a door open for us to use them in future enhancements` - we could just add $type as an optional param later on if we need it?

Reply:
I'll just delete it because you'll never agree with me and we no longer have a third person to act as arbiter.